### DISCUSSION TITLE
Grand potential Phase field model coupled with mechanics

### DISCUSSION THREAD

**Discussion by mishkat1096** [idaholab/moose#20751] [Q&A Modules: Phase field] [answered]
Hi,
I am trying to use Grand potential phase-field model to simulate a binary phase material. I want to include the elastic energy from the interface in the calculation.
Is there a similar example to begin with? 
I have seen mechanics coupled KKS model but not grand potential model.

Thanks
Mishkat

**Comment by GiudGiud** [comment, marked as answer]
Hello

You can have a look at this test
https://github.com/idaholab/moose/blob/next/modules/phase_field/test/tests/phase_field_kernels/ACInterfaceStress.i
@SudiptaBiswas do you know of another example?

Guillaume

**Reply by SudiptaBiswas** [reply]
@mishkat1096 You probably should take a look at this paper first, https://doi.org/10.1016/j.commatsci.2020.109790

**Reply by mishkat1096** [reply]
Thanks for the help.

**Reply by THOMPSONFLORIDA** [reply]
@mishkat1096 
please can i have a copy of the kks phase field coupled with elastic energy
thompsonigunma@ufl.edu
